prost-build/src/lib.rs

```diff
@@ -131,37 +131,34 @@
 //!
 //! [`protobuf-src`]: https://docs.rs/protobuf-src
 
-use std::collections::HashMap;
-use std::default;
-use std::env;
-use std::ffi::{OsStr, OsString};
 use std::fmt;
-use std::fs;
-use std::io::{Error, ErrorKind, Result, Write};
+use std::io::Result;
 use std::ops::RangeToInclusive;
-use std::path::{Path, PathBuf};
-use std::process::Command;
+use std::path::Path;
 
-use log::debug;
-use log::trace;
-
-use prost::Message;
-use prost_types::{FileDescriptorProto, FileDescriptorSet};
+use prost_types::FileDescriptorSet;
 
+mod ast;
 pub use crate::ast::{Comments, Method, Service};
-use crate::code_generator::CodeGenerator;
-use crate::extern_paths::ExternPaths;
-use crate::ident::to_snake;
-use crate::message_graph::MessageGraph;
-use crate::path::PathMap;
 
-mod ast;
 mod code_generator;
 mod extern_paths;
+
 mod ident;
+use crate::ident::to_snake;
+
 mod message_graph;
 mod path;
 
+mod config;
+// HELP(gibbz00): I think `error_message_protoc_not_found`, `protoc_from_env`, and `protoc_include_from_env`
+// should be made private. Only used inside config and not in any other part
+// of prost. Leaving this as is for now as this would technically be breaking
+// changes.
```

Comment:
I agree these should be private. I am willing to accept this breaking change. If you want to prevent breaking changes, then this should change to a `TODO(breaking-change):` comment.

Reply:
Sweet. I'll keep that in mind next time :)